common/src/IO/TextureLoader.cpp

```diff
@@ -58,6 +59,9 @@ namespace TrenchBroom {
             } else if (textureConfig.format.format == "idwal") {
                 TextureReader::PathSuffixNameStrategy nameStrategy(2, true);
                 return new IdWalTextureReader(nameStrategy, loadPalette(textureConfig));
+            } else if (textureConfig.format.format == "freeimage") {
```

Comment:
I would prefer it if this would just be called "image" instead. Noone needs to know how the loader is implemented, so why name it "freeimage"?